"es" to "de" translation has changed

Spanish to German is now "mein name ist jeff"

#### Steps to reproduce

  1. test python-translate/tests/system.py 

#### Stack trace
```
    def test_translate(self):
        values = ["petnaest", "dek kvin", "Me llamo Jeff", "My name is Jeff"]
        translations = Config.CLIENT.translate(
            values, target_language="de", model="nmt"
        )
        self.assertEqual(len(values), len(translations))
    
        self.assertEqual(translations[0]["detectedSourceLanguage"].lower(), "hr")
        self.assertEqual(translations[0]["translatedText"].lower(), u"fünfzehn")
    
        self.assertEqual(translations[1]["detectedSourceLanguage"], "eo")
        self.assertEqual(translations[1]["translatedText"].lower(), u"fünfzehn")
    
        self.assertEqual(translations[2]["detectedSourceLanguage"], "es")
>       self.assertEqual(translations[2]["translatedText"].lower(), u"ich heiße jeff")
E       AssertionError: 'mein name ist jeff' != 'ich heiße jeff'
E       - mein name ist jeff
E       + ich heiße jeff
```

--- a/noxfile.py
+++ b/noxfile.py
@@ -138,7 +138,7 @@ def docs(session):
     """Build the docs for this library."""
 
     session.install("-e", ".")
-    session.install("sphinx", "alabaster", "recommonmark")
+    session.install("sphinx<3.0.0", "alabaster", "recommonmark")
 
     shutil.rmtree(os.path.join("docs", "_build"), ignore_errors=True)
     session.run(

--- a/tests/system.py
+++ b/tests/system.py
@@ -67,7 +67,8 @@ class TestTranslate(unittest.TestCase):
         self.assertEqual(translations[1]["translatedText"].lower(), u"fünfzehn")
 
         self.assertEqual(translations[2]["detectedSourceLanguage"], "es")
-        self.assertEqual(translations[2]["translatedText"].lower(), u"ich heiße jeff")
+        es_translation = translations[2]["translatedText"].lower()
+        self.assertTrue(es_translation==u"ich heiße jeff" or es_translation==u"mein name ist jeff")
 
         self.assertEqual(translations[3]["detectedSourceLanguage"], "en")
         self.assertEqual(

--- a/tests/system.py
+++ b/tests/system.py
@@ -68,7 +68,10 @@ class TestTranslate(unittest.TestCase):
 
         self.assertEqual(translations[2]["detectedSourceLanguage"], "es")
         es_translation = translations[2]["translatedText"].lower()
-        self.assertTrue(es_translation==u"ich heiße jeff" or es_translation==u"mein name ist jeff")
+        self.assertTrue(
+            es_translation == u"ich heiße jeff"
+            or es_translation == u"mein name ist jeff"
+        )
 
         self.assertEqual(translations[3]["detectedSourceLanguage"], "en")
         self.assertEqual(

--- a/tests/unit/v2/test__http.py
+++ b/tests/unit/v2/test__http.py
@@ -86,5 +86,9 @@ class TestConnection(unittest.TestCase):
         }
         expected_uri = conn.build_api_url("/rainbow")
         http.request.assert_called_once_with(
-            data=req_data, headers=expected_headers, method="GET", url=expected_uri,
+            data=req_data,
+            headers=expected_headers,
+            method="GET",
+            url=expected_uri,
+            timeout=60,
         )

--- a/tests/unit/v2/test__http.py
+++ b/tests/unit/v2/test__http.py
@@ -80,15 +80,11 @@ class TestConnection(unittest.TestCase):
         self.assertEqual(result, data)
 
         expected_headers = {
+            "Accept-Encoding": "gzip",
             base_http.CLIENT_INFO_HEADER: conn.user_agent,
             "User-Agent": conn.user_agent,
-            "Accept-Encoding": "gzip",
         }
         expected_uri = conn.build_api_url("/rainbow")
         http.request.assert_called_once_with(
-            data=req_data,
-            headers=expected_headers,
-            method="GET",
-            url=expected_uri,
-            timeout=None,
+            data=req_data, headers=expected_headers, method="GET", url=expected_uri,
         )

